core/types/external_tx.go

```diff
@@ -27,6 +27,13 @@ type ExternalTx struct {
 	// the origin chain indeed confirmed emission of that ETX.
 }
 
+// PendingEtxsRollup is Header and manifest Hash of that header that should
+// be forward propagated
+type PendingEtxsRollup struct {
+	Header *Header       `json:"header" gencodec:"required"`
+	Rollup []common.Hash `json:"rollup" gencodec:"required"`
```

Comment:
Would prefer to call this Manifest and use the `BlockManifest` type

Reply:
Using BlockManifest type now

Reply:
will make sure to check if i am using slice of common  hash anywhere for the rollup